apps/coscrad-frontend/src/utils/generic-components/presenters/detail-views/resource-detail-thumbnail-presenter.tsx

```diff
@@ -30,16 +30,17 @@ export const ResourceDetailThumbnailPresenter = ({
     <Card>
         <CardContent>
             <Grid container spacing={1} columns={{ xs: 2, sm: 4, md: 12 }}>
+                <div data-testid={id} />
                 <Grid item xs={2} sm={1} md={2}>
                     {/* Preview will eventually include images taken from video or photos, etc. */}
                     <ResourcePreviewIconFactory resourceType={type} size="md" />
                 </Grid>
-                <Grid item xs={2} sm={2} md={8}>
+                <Grid item xs={2} sm={2} md={7}>
                     {/* TODO: consider putting a standardized name property on the view models */}
-                    <ResourceNamePresenter name={name} variant="h5" />
+                    <ResourceDetailPresenterHeader id={id} type={type} name={name} variant="h5" />
```

Comment:
While I'm here I thought I'd try out the info icon with a tooltip that shows the ID for the resource.  Apologies for scope creep...